app/build.gradle

```diff
@@ -48,6 +48,8 @@ dependencies {
             appDependencies.cardView,
             appDependencies.material
 
+    implementation 'androidx.appcompat:appcompat:1.0.2'
```

Comment:
remove this and add the dependencies to the generalDependencies, just like the others pls

Reply:
done